### ISSUE TITLE
Doesn't work in mobile

### ISSUE THREAD

**Issue by MuhammadBayiz** [magicoflolis/Magic-PH#1]
I'm using Kiwi Browser Latest version and the userscript didn't work showing this error in the console
![Screenshot_20220917-231829_Kiwi Browser](https://user-images.githubusercontent.com/63660736/190875140-f089f089-f57e-403f-b113-e70a7eccd6f9.png)

**Comment by magicoflolis** [comment]
I was able to load it in the latest available Kiwi browser using TamperMonkey, the mobile bookmarklet doesn't seem to work on this browser though. It also seems like the script is being blocked by another task/extension.
Android Emulator: Nox
Android version: 7.1.2

* Also try toggling "Desktop site"

![image](https://user-images.githubusercontent.com/10752602/193435160-3abcc186-1bc3-488d-954c-a22a9d40da20.png)

**Comment by MuhammadBayiz** [comment]
Unfortunately in a smart phone it isn't like that when I'm on mobile phone view the script doesn't work when i switch to desktop view the video players control doesn't show because when i touch the video player it takes me to Fullscreen and again touching it it exits the Fullscreen you can't see the video player control at all to get to the quality selection page

**Comment by MuhammadBayiz** [comment]
And here's a screen recording
https://photos.app.goo.gl/5TjsBYNgDsFQUFG39

**Comment by magicoflolis** [comment]
> This line fetches the current page and locates the videos qualities from within the page (on PornHub), each site differs slightly.
https://github.com/magicoflolis/Magic-PH/blob/8c00b9f800f77f4a9ab4ce2eb817b619ef466a51/userscript/dist/magicph.user.js#L261

Looking back @ your first screenshot, it does seem the UserJS is unable to fetch the current page. (i.e., https://www.pornhub.com/view_video.php...) There could also be more @ play here as well.

Perhaps try some alternatives:

* Alternative UserJS Manager(Violentmonkey): https://chrome.google.com/webstore/detail/violentmonkey/jinjaccalgkegednnccohejagnlnfdag
* More in depth adblocker(uBlock Origin): https://chrome.google.com/webstore/detail/ublock-origin/cjpalhdlnbpafiamejdnhcphjbkeiagm
* If you wish to try a different web browser. Try Firefox or Bromite which has built-in adblock + userscript support, but can't load chrome extensions: https://www.bromite.org/

In addition, I have a Samsung Galaxy Note 10+ running Android 12 and I used this Kiwi Browser https://play.google.com/store/apps/details?id=com.kiwibrowser.browser

| Screenshot |
|:----------:|
![SmartSelect_20221003_162002_Kiwi Browser](https://user-images.githubusercontent.com/10752602/193703293-750a75d5-101c-4b3a-abe7-3b9595594255.jpg)

**Comment by MuhammadBayiz** [comment]
I tried other userscript manager but the same error in fetching (couldn't figure it out what seems to be the problem) I re installed both ublock origin and TamperMonkey again no luck so finally i uninstalled Kiwi Browser and re download it again in Play Store somehow that fixed the issue, probably leftover files from old version of the browser. Anyway now it's working and everything is pretty fast and efficient thanks for the awesome userscript 👍😎

**Comment by MuhammadBayiz** [comment]
I figured it out! You just need to disable the "Hide annoying ads"
![Screenshot_20221005-155806_Kiwi Browser](https://user-images.githubusercontent.com/63660736/194066441-c2859cf4-6118-49ee-a1e4-f623f77f51b7.png)
 i think it's blocking the userscript from fetching the urls as it appears in this error pop up
![Screenshot_20221005-155815_Kiwi Browser](https://user-images.githubusercontent.com/63660736/194066497-e23ee745-e584-4836-93b6-93a133e36799.png)
